templates/rbac.yaml

```diff
@@ -68,17 +26,11 @@ rules:
       - "" # indicates the core API group
       - "apps"
     resources:
-      # Every environment deployment has a service.
-      - services
```

Comment:
We still use services, so this cannot be deleted, the wording could be changed to `Every workspace has a service` though.

Reply:
Ahh. Once the new network stuff comes to fruition I think we can actually remove this too :o